plugins/available/systemd.plugin.bash

```diff
@@ -0,0 +1,50 @@
+# Author: Arthur Miller
+# Created: 2017-08-08
+# License: Public Domain
+
+cite about-plugin
+about-plugin 'SystemD helper functions'
+
+# Reloads a systemwide service. If no agruments are given, reloads all services.
+# Requires admin priviledge.
+scrs(){
+    if [ $# -eq 0 ]; then
+	systemctl daemon-reload
```

Comment:
The indentation seems to be off for these lines. Can you please indent the lines with 2 spaces per level? The same goes for the other functions - they all seem to be off...